### ISSUE TITLE
[BUG] Sinn des Quickstart-Guides

### ISSUE THREAD

**Issue by cagix** [PM-Dungeon/desktop#73]
Was ist der Mehrwert, dass man sich das Zeug aus dem Quickstart-Guide schrittweise rauskopiert? Wäre eine Code-Vorgabe nicht sinnvoller und würde irgendwelche Importprobleme vermeiden (manche Klassen gibt es an mehreren Stellen!)? 

Der Guide selbst ist schon hilfreich, aber ich sehe kaum Mehrwert darin, die Codeschnipsel schrittweise rauszukopieren?

**Comment by AMatutat** [comment]
Es zwingt dich den Quickstartguide wirklich zu lesen.

**Comment by cagix** [comment]
> Es zwingt dich den Quickstartguide wirklich zu lesen. 
> 
> 

stimmt. und mich dann über die struktur, die formulierungen und die vielen fehler zu ärgern.

**Comment by AMatutat** [comment]
schön das wir das geklärt haben. 
Ich vertrete die Auffassung, eine Tour durch fertig geschrieben Code ist deutlich weniger effizient als angeleitetes Code schreiben.

**Comment by cagix** [comment]
> schön das wir das geklärt haben. 
> 
> Ich vertrete die Auffassung, eine Tour durch fertig geschrieben Code ist deutlich weniger effizient als angeleitetes Code schreiben. 

ja, bin ich bei dir. nur in der form versteht man grad noch nix und kopiert stumpf die anweisungen raus. da die teilweise fehlerhaft sind und zusätzlich weitere fehler beim übertragen passieren, sehe ich das ziel nicht erreicht.
